mmcv/runner/default_constructor.py

```diff
@@ -0,0 +1,17 @@
+from .builder import RUNNER_BUILDERS, RUNNERS
+
+
+@RUNNER_BUILDERS.register_module()
+class DefaultRunnerConstructor:
+    """Default constructor for runners.
+    """
```

Comment:
[Nit]: update the docstring of this module

Reply:
add examples. Thanks @zhouzaida